docs/content/Checkbox.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Checkbox
+status: Alpha
+source: https://github.com/primer/react/blob/main/src/Checklist.tsx
+storybook: '/react/storybook?path=/story/forms-checkbox--default'
+---
+
+import {ComponentChecklist} from '../src/component-checklist'
+
+`Checkbox` is a form component that applies Primer design language to a native HTML checkbox input element.
```

Comment:
Could we move this into the `description` frontmatter and reword it to be a little more actionable (see other [component descriptions](https://github.com/primer/component-metadata/blob/main/metadata.json)):

```
---
title: Checkbox
description: Use checkboxes to ...
---
```

Reply:
Moved and content updated 👍